plugins/inputs/opcua/README.md

```diff
@@ -47,34 +47,57 @@ Plugin minimum tested version: 1.16
   # password = ""
   #
   ## Node ID configuration
-  ## name       			- the variable name
-  ## namespace  			- integer value 0 thru 3
-  ## identifier_type		- s=string, i=numeric, g=guid, b=opaque
-  ## identifier			- tag as shown in opcua browser
-  ## data_type  			- boolean, byte, short, int, uint, uint16, int16,
-  ##                        uint32, int32, float, double, string, datetime, number
+  ## field_name        - field name to use in the output
```

Comment:
this documentation is a bit confusing; I thought these were top-level settings until I got to the example below. Not sure what we can do to improve it.

Reply:
Nodes was a top level setting before this PR.  I added groups in this PR and each group also has a set of nodes.  I left in the top level nodes for backward compatibility.  I can see that this is confusing so I'll have to work on the docs.